src/commands/publish.js

```diff
@@ -8,48 +9,56 @@ import Project from '../Project';
 
 export type PublishOptions = {|
   cwd?: string,
+  access?: string,
 |};
 
 export function toPublishOptions(args: options.Args, flags: options.Flags): PublishOptions {
   return {
     cwd: options.string(flags.cwd, 'cwd'),
+    access: options.string(flags.access, 'access')
```

Comment:
Maybe you'll be able to answer this, but when I first tried to publish I got errors because it was a scoped package so you have to manually add the --access public flag. After it was published though, it didnt need that flag anymore? Is it only required on initial publish?